src/plugins/interactive_setup/public/format_bash_command.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export function formatBashCommand(command: string, args?: string) {
+  const isWindows = window.navigator.userAgent.includes('Win');
+  return `${isWindows ? `bin\\${command}.bat` : `bin/${command}`}${args ? ` ${args}` : ''}`;
```

Comment:
__note:__ it's great that we have this code in one place now :tada: If users ever complain about false positives here we'll need to update just one place (e.g. fetch actual OS from the server).